README.md

```diff
@@ -38,13 +38,20 @@ $ geth --help
     --builder                      (default: false)
           Enable the builder
 
+    --builder.algotype value       (default: "mev-geth")
+          Block building algorithm to use [=mev-geth] (mev-geth, greedy, greedy-buckets)
+   
     --builder.beacon_endpoints value (default: "http://127.0.0.1:5052")
           Comma separated list of beacon endpoints to connect to for beacon chain data
           [$BUILDER_BEACON_ENDPOINTS]
 
     --builder.bellatrix_fork_version value (default: "0x02000000")
           Bellatrix fork version. [$BUILDER_BELLATRIX_FORK_VERSION]
 
+    --builder.blacklist value     
```

Comment:
useful to add that this also affects block validation API

Reply:
good catch, updated